HACKING.md

```diff
@@ -0,0 +1,25 @@
+# Hacking tips for Akira
```

Comment:
Personally, feel `Hacking` is not a good term to be used here.

**Hacking** has a negative vibe to it. Plus what is being talked about here is mostly how to setup the build environment so it matches the repo and the team standards.

Consider renaming it `Tips for building Akira`? File name could be `BUILD.md`

Reply:
We all are hackers, and we hack on code. 

https://www.dourish.com/goodies/jargon.html

```
HACKER [originally, someone who makes furniture with an axe] n. 1. A person who enjoys learning the details of programming systems and how to stretch their capabilities, as opposed to most users who prefer to learn only the minimum necessary. 2. One who programs enthusiastically, or who enjoys programming rather than just theorizing about programming. 3. A person capable of appreciating hack value (q.v.). 4. A person who is good at programming quickly. Not everything a hacker produces is a hack. 5. An expert at a particular program, or one who frequently does work using it or on it; example: "A SAIL hacker". (Definitions 1 to 5 are correlated, and people who fit them congregate.) 6. A malicious or inquisitive meddler who tries to discover information by poking around. Hence "password hacker", "network hacker".
```

of course we can rename if everyone think is a better name. I would suggest `CONTRIBUTING.md`

Reply:
Yeah, agree, but hacker word in itself is always associated in a negative sense. That is why I was suggesting something else.

My reasoning is that the project should be more welcoming than try to be too nerdy for others. When we use such words, a lot of the first time contributors would not even attempt anything thinking it is something complex. 

`CONTRIBUTING.md` I think is also the standard on Github.

Reply:
Let's rename to `CONTRIBUTING.md`

Reply:
Done